SystemSprawozdan/SystemSprawozdan.Frontend/Pages/Student/Subjects/SignUpToGroup.razor

```diff
@@ -79,7 +79,7 @@
                 };
             await AppHttpClient.Post("SubjectSubgroup", subgroup);
             Toaster.Add("Poprawnie stworzono grupę", MatToastType.Success, "Sukces");
-            NavigationManager.NavigateTo("/teams");
+            NavigationManager.NavigateTo("Student/subjects");
```

Comment:
jak kazdego if'a konczysz tak samo to umiesc to za nim ale w tej stuacji zamiast robic przekierowanie mozesz poprostu pobrac ponownie dane z bazy

Reply:
Spoko ale ja zamieniłem tego linka co był na na tego do mojej strony xD